pkg/cfgfile/cfgfile.go

```diff
@@ -31,11 +31,76 @@ import (
 	"github.com/xeipuuv/gojsonschema"
 )
 
+// we're wrapping a function from the underlying io/ioutil package here so
+// that it's easier to mock when we define our *small* tests for this package...
+// first, define an interface for reading a file (by path)
+type reader interface {
+	ReadFile(s string) ([]byte, error)
```

Comment:
Idk how big of an issue it is, maybe someone else will chime in. My first thought was why are we redefining a well known interface.